requirements.txt

```diff
@@ -1,7 +1,27 @@
-editquality >= 0.2.1
-ores >= 0.5.7
-redis
-statsd
-yamlconf >= 0.1.0
-wb-vandalism >= 0.1.6
-wikiclass >= 0.2.1
+Flask==0.10.1
+Flask-Jsonpify==1.5.0
+Jinja2==2.8
+PyYAML==3.11
+Werkzeug==0.11.4
+celery==3.1.22
+docopt==0.6.2
+editquality==0.3.0
+itsdangerous==0.24
+mwapi==0.3.1
+mwreverts==0.0.6
+mwtypes==0.2.0
+mwxml==0.2.1
+numpy==1.10.4
+ores==0.5.7
+python-Levenshtein==0.12.0
+pytz==2012c
+pywikibase==0.0.4
+redis==2.10.5
+requests==2.9.1
+revscoring==1.0.1
```

Comment:
Shouldn't be 1.0.1 yet.  This is due to an issue in editquality.  Switch this to 0.7.11 for now.